spellcheck/spellcheck_utils.cpp

```diff
@@ -6,7 +6,7 @@
 //
 #include "spellcheck/spellcheck_utils.h"
 
-#include <QtCore/QLocale>
+#include "spellcheck/platform/platform_spellcheck.h"
```

Comment:
Nitpick: I prefer to separate general `include`s from Qt `include`s with an empty line.